cvat/apps/documentation/user_guide.md

```diff
@@ -241,10 +251,58 @@ Usage examples:
 
```

Comment:
*Semi-automatic* annotation is annotation which have been created mainly automatically, but user have provided some data for it. In my opinion. For example: interpolation.